build/build.sh

```diff
@@ -12,7 +12,8 @@ export TZ=Etc/UTC
 export LC_ALL=en_US.UTF-8
 
 # Log the external-resources commit used when building the manuscript
-EXTERNAL_RESOURCES_COMMIT=$(curl -sS https://api.github.com/repos/greenelab/covid19-review/branches/external-resources | python -c "import sys, json; print(json.load(sys.stdin)['commit']['sha'])")
+curl -sS https://api.github.com/repos/greenelab/covid19-review/branches/external-resources > commitinfo.json
```

Comment:
```suggestion
curl -sS --retry 3 --retry-max-time 30 https://api.github.com/repos/greenelab/covid19-review/branches/external-resources > commitinfo.json
```
Should we add a retry? I'm not sure what the problem is or if this will help.